<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EditVehicleDialog</class>
 <widget class="QDialog" name="EditVehicleDialog">
  <property name="windowTitle">
   <string>Editar Vehículo</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout">
     <item>
      <widget class="QLabel" name="licensePlateLabel">
       <property name="text">
        <string>Patente:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="licensePlateLineEdit">
       <property name="placeholderText">
        <string>Placa</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout">
     <item>
      <widget class="QLabel" name="typeLabel">
       <property name="text">
        <string>Tipo de vehículo:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="typeLineEdit">
       <property name="placeholderText">
        <string>Tipo</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout">
     <item>
      <widget class="QLabel" name="colorLabel">
       <property name="text">
        <string>Color:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="colorLineEdit">
       <property name="placeholderText">
        <string>Color</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout">
     <item>
      <widget class="QLabel" name="brandLabel">
       <property name="text">
        <string>Marca:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="brandLineEdit">
       <property name="placeholderText">
        <string>Marca</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout">
     <item>
      <widget class="QLabel" name="modelLabel">
       <property name="text">
        <string>Modelo:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="modelLineEdit">
       <property name="placeholderText">
        <string>Modelo</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Ok|QDialogButtonBox::Cancel</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
